src/lib/containers/EntityBadgeIcons.tsx

```diff
@@ -149,7 +149,6 @@ export const EntityBadgeIcons = (props: EntityBadgeIconsProps) => {
   useEffect(() => {
     if (isInExperimentalMode && schemaValidationResults) {
       // Because we may not know to render an icon until after fetching data (i.e. after a delay), we have to rebuild the tooltip to ensure it appears
```

Comment:
This comment is now either inaccurate (and should be removed), or we need to find another way to rebuild the tooltip after the data has been fetched